projects/react/package.json

```diff
@@ -30,13 +30,13 @@
         "Georgiy Lunin <nextzeddicus@gmail.com>"
     ],
     "devDependencies": {
-        "@testing-library/react": "12.1.5",
+        "@testing-library/react": "13.4.0",
         "@testing-library/react-hooks": "8.0.1",
         "@testing-library/user-event": "14.4.3",
         "@types/react": "18.2.21",
         "@types/react-dom": "18.2.7",
-        "react": "17.0.2",
-        "react-dom": "17.0.2",
+        "react": "18.2.0",
```

Comment:
**Note:**
it is `devDependencies` (not `peerDependencies`!). 
It is used for unit testing only.